egs/MedleyDB/ConvTasNet/run.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
Fix this file. 
- Remove all the unused variables
- Indent correctly
- Add instructions to download MedleyDB